test/test_images/sequencestepper/main.go

```diff
@@ -61,7 +63,11 @@ func main() {
 	// parse the command line flags
 	flag.Parse()
 
-	t, err := cloudevents.NewHTTP(cloudevents.WithPort(8080))
+	t, err := cloudevents.NewHTTP(
+		cloudevents.WithPort(8080),
+		cloudevents.WithRoundTripper(&ochttp.Transport{
```

Comment:
This image uses only the receiver, so you don't need `WithRoundTripper`, because it's used only for the underlying HTTP client, for the server side you need to use `WithMiddleware` (you could easily reuse `kncloudevents.CreateHandler` for this)

Reply:
Done.